Helper/Activity.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Metrilo\Analytics\Helper;
+
+class Activity extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    private $endPoint = 'http://p.metrilo.com';
+    
+    public function createActivity($storeId, $type, $token, $secret)
+    {
+        
+        $data = array(
+            'type'          => $type,
+            'project_token' => $token,
+            'signature'     => md5($token . $type . $secret)
+        );
+        
+        $url = $this->endPoint . '/tracking/' . $token . '/activity';
```

Comment:
What is the point in separating the hostname in the `endPoint` variable?

Reply:
Actually there is no point in placing it in variable. Removed.

Reply:
is it fixed now? what about QA envs?

Reply:
As far as i can understand QA envs and production envs should both send activity calls to p.metrilo.com

Reply:
They have completely different DBs and the calls will fail

Reply:
So production env should send to p.metrilo.com and QA env should send to qa.env.tracking ?

Reply:
If the env name is plg-124 in QA it would be: `monolith-web-plg-124.customerboard.co`

Reply:
I see, it was done that way using $endPoint = $this->getApiEndpoint() where endpoint will depend on the installation (
bin/magento config:set metrilo_analytics/general/api_endpoint http://p.metrilo.com - for production  AND
bin/magento config:set metrilo_analytics/general/api_endpoint $NEW_TRACKING_ENDPOINT - for QA )